website/index/samples/sample.kotlin.txt

```diff
@@ -0,0 +1,28 @@
+﻿const val POINTS_X_PASS: Int = 15
+val EZPassAccounts: MutableMap<Int, Int> = mutableMapOf(1 to 100, 2 to 100, 3 to 100)
+val EZPassReport: Map<Int, Int> = EZPassAccounts
+
+// update points credit
+fun updatePointsCredit(accountId: Int) {
+    if (EZPassAccounts.containsKey(accountId)) {
+        println("Updating $accountId...")
+        EZPassAccounts[accountId] = EZPassAccounts.getValue(accountId) + POINTS_X_PASS
+    } else {
+        println("Error: Trying to update a non-existing account (id: $accountId)")
+    }
+}
+
+fun accountsReport() {
+    println("EZ-Pass report:")
+    EZPassReport.forEach{

```

Comment:
Follow the kotlin code style, please put a space before the brace after foreach.